libcxx/test/std/input.output/iostream.format/output.streams/ostream.formatted/ostream.formatted.print/locale-specific_form.pass.cpp

```diff
@@ -95,6 +96,16 @@ test(std::stringstream& stream, std::string expected, test_format_string<char, A
                  TEST_WRITE_CONCATENATED(
                      "\nFormat string   ", fmt.get(), "\nExpected output ", expected, "\nActual output   ", out, '\n'));
   }
+  // *** println blank line***
+  {
+    expected = "\n";
+    stream.str("");
+    ;
```

Comment:
```suggestion
  
```
Can you also fix this on line 71 and 81?

Reply:
Sure!